deep_quoridor/src/agents/alphazero/self_play_manager.py

```diff
@@ -64,6 +63,14 @@ def run(self):
         else:
             self._run_central_evaluation()
 
+    # Return an array with the number of games per worker, such that they add up to the total number of games
+    # and each worker gets X or X+1 jobs.
+    # E.g. for 20 games and 8 workers it returns: [3, 3, 3, 3, 2, 2, 2, 2]
+    def _games_per_worker(self):
+        base = self.num_games // self.num_workers
+        extra = self.num_games - base * self.num_workers
+        return [base + 1 if i < extra else base for i in range(self.num_workers)]
+
```

Comment:
I noticed when playing 20 games with 8 workers that it was assigning 3 per worker, so it was playing 24 games.
Then I realized it was missing `games_remaining_to_allocate -= this_worker_num_games`, but this didn't work as I expected, since in that case it was doing 6 workers with 3 games, 1 worker with 2 and another worker empty.

Reply:
I definitely need to do a better job of debugging before making PR's :-( Sorry!